common/lua-system.cpp

```diff
@@ -7,24 +7,25 @@
 
 #include "entity.hpp"
 
-namespace tec {	
+namespace tec {
 	void LuaSystem::Update(const double delta) {
 		auto _log = spdlog::get("console_log");
 		ProcessCommandQueue();
-		
+
 		for (auto itr = this->scripts_map.Begin(); itr != this->scripts_map.End(); itr++) {
 			auto entity_id = itr->first;
 			if (Entity(entity_id).Has<LuaScript>()) {
 				if (const LuaScript* lscript = Entity(entity_id).Get<LuaScript>()) {
-					if (!lscript->script_name.empty() ) { // Check that have a assigned script with onUpdate function
-						//lscript->state["this"].SetObj(Entity(entity_id) //,
-							//"add", &Entity::Add,
-							//"has", &Entity::Has,
+					if (!lscript->script_name
+							 .empty()) { // Check that have a assigned script with onUpdate function
```

Comment:
I would much much rather put the comment on the previous line than split the line like this. in fact, i think end-of-line comments are a bad idea in general.

Reply:
Yeah, not much the tool can do to fix that completely, but it could be part of the review process to move large comments to next line. Tiny comments for end of line (e.g. `auto time = 1; // in seconds` are good, but this is definitely not a case where it is appropriate.

Reply:
IMO those comments should also be on their own line.